Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/AzureModule.java

```diff
@@ -138,6 +141,11 @@ protected void refreshItems() throws AzureCmdException {
         if (hdInsightModule != null && !isDirectChild(hdInsightModule)) {
             addChildNode(hdInsightModule);
         }
+
+        if (sparkServerlessClusterRootModule != null && !isDirectChild(sparkServerlessClusterRootModule)) {
```

Comment:
We need to add a feature switch here. Consider to use Java system property to enable the feature.